packages/expo-device/src/DeviceWithNotchConst.ts

```diff
@@ -0,0 +1,226 @@
+export const devicesWithNotch = [
```

Comment:
@brentvatne will there be a better way to know if the device has a notch with LayoutContext (or whatever Janic's work is called)? Seems important too if a device has multiple screens and we want to know if the current screen has a notch.